modules/ossm-security-cert-manage.adoc

```diff
@@ -29,7 +29,7 @@ $ oc create secret generic cacerts -n istio-system --from-file=<path>/ca-cert.pe
     --from-file=<path>/cert-chain.pem
 ----
 +
-. In the `ServiceMeshControlPlane` resource set `spec.security.dataPlane.mtls: true` to `true` and configure your certificateAuthority like the following example. The default `rootCADir` is `/etc/cacerts`. You do not need to set the `privateKey` if the key and certs are mounted in the default location.  {ProductShortName} reads the certificates and key from the secret-mount files.
+. In the `ServiceMeshControlPlane` resource set `spec.security.dataPlane.mtls` to `true` and configure the `certificateAuthority` field like the following example. The default `rootCADir` is `/etc/cacerts`. You do not need to set the `privateKey` if the key and certs are mounted in the default location.  {ProductShortName} reads the certificates and key from the secret-mount files.
```

Comment:
@jwendell the only recommendation I'd make is to change "...like the following example" to "...as shown in the following example"